### ISSUE TITLE
Master broken by zypper conflict between kmod and kmod-compat

### ISSUE THREAD

**Issue by jayvdb** [coala/docker-coala-base#210]
```
Checking for file conflicts: [...........error]

Detected 9 file conflicts:

File /usr/share/man/man5/depmod.d.5.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man5/modprobe.d.5.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man5/modules.dep.5.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/depmod.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/insmod.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/lsmod.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/modinfo.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/modprobe.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File /usr/share/man/man8/rmmod.8.gz
  from install of
     kmod-24-1.1.x86_64 (OSS)
  conflicts with file from package
     kmod-compat-23-2.3.x86_64 (@System)

File conflicts happen when two packages attempt to install files with the same name but different contents. If you continue, conflicting files will be replaced losing the previous content.

Continue? [yes/no] (no): no

Problem occurred during or after installation or removal of packages:

Installation aborted by user

Please see the above error message for a hint.
```

https://travis-ci.org/coala/docker-coala-base/builds/253587305

https://cloud.docker.com/swarm/coala/repository/docker/coala/base/builds/ee31198c-41ea-478c-aa1b-3d65c6d9b517

**Comment by jayvdb** [comment]
Worth noting, these conflicts likely are not even 'real' conflicts, as these manpages shouldnt be installed due to `rpm.install.excludedocs = yes`.

**Comment by jayvdb** [comment]
As this is upstream, noting that the current (9ac1732) Dockerfile is also broken by https://github.com/coala/docker-coala-base/issues/211, which can be fixed by applying a9cd6832d first.

**Comment by jayvdb** [comment]
Reported upstream as https://bugzilla.opensuse.org/show_bug.cgi?id=1048835
